packages/react-api/src/overrides/spec/kulupu.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2020 @polkadot/react-api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default {
+  Difficulty: 'U256',
+  DifficultyAndTimestamp: {
+    difficulty: 'Difficulty',
+    timestamp: 'Moment'
+  },
+  DispatchError: 'DispatchErrorTo198',
+  DispatchResult: 'DispatchResultTo198',
+  DispatchInfo: {
+    weight: 'Weight',
+    class: 'DispatchClass'
+  }
```

Comment:
cc @sorpaas - There can be removed once bumped to latest Substrate master

Reply:
Just a small note, the bump will probably take several months due to the infrequent fork schedule. (It currently uses a [hard fork module](https://github.com/kulupu/kulupu/blob/master/anyupgrade/src/lib.rs) and has not yet transitioned to on-chain voting.)

Reply:
No issues, I saw that. No issues, the types are there, nothing will break in the meantime, we should just keep it somewhere in the memory banks to update once there are changes.